GIGA AREF info [HC-1521] (#357)
arduino/help-center-content: content/Hardware Support/Generic/About-the-AREF-pin.md

      <blockquote><b>Note:</b> To avoid damaging your board, read the library reference's <a href="https://www.arduino.cc/reference/en/language/functions/analog-io/analogreference/#_notes_and_warnings">notes and warnings</a> section before using an external reference voltage.</blockquote>
    </td>
  </tr>
  <tr>
    <td>
      <ul>

>         <li>GIGA</li>

Reviewer: ```suggestion
        <li>Arduino GIGA R1</li>
```

      </ul>
    </td>
    <td>

>         The analog reference is automatically selected when the AREF pin is connected to a voltage.
>         The GIGA doesn’t need to use analogReference() to indicate the reference since it automatically changes the state when a voltage is added.

Reviewer: ```suggestion
        Any external voltage applied to the AREF pin will be automatically used as the reference.
```
The behavior appears to be the same as for Portenta H7, which is also included in this table.

We can use the same description here for consistency.
